sdk/core/core/src/az_curl_adapter.c

```diff
@@ -162,9 +162,22 @@ az_curl_send_post_request(CURL * const p_curl, az_http_request_builder const * c
   AZ_CONTRACT_ARG_NOT_NULL(p_hrb);
 
   // Method
-  AZ_RETURN_IF_CURL_FAILED(curl_easy_setopt(p_curl, CURLOPT_POSTFIELDS, p_hrb->body.begin));
+  az_mut_span body = { 0 };
+  AZ_RETURN_IF_FAILED(az_span_malloc(p_hrb->body.size + AZ_STR_ZERO.size, &body));
 
-  AZ_RETURN_IF_CURL_FAILED(curl_easy_perform(p_curl));
+  az_mut_span zt_buf = { 0 };
+  az_result res_code = az_mut_span_to_str(body, p_hrb->body, &zt_buf);
+
+  if (az_succeeded(res_code)) {
```

Comment:
I get it, it's like actions but more code https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/core/core/src/az_curl_slist.c#L39 ;-) Just kidding.